fuel-core/src/database/block.rs

```diff
@@ -139,6 +139,22 @@ impl Database {
             })
     }
 
+    pub fn genesis_block_ids(&self) -> Result<(BlockHeight, Bytes32), Error> {
+        self.iter_all(
+            Column::FuelBlockIds,
+            None,
+            None,
+            Some(IterDirection::Forward),
+        )
+        .next()
+        .ok_or(not_found!("Genesis block height"))?
+        .map(|(height, id): (Vec<u8>, Bytes32)| {
+            // safety: we know that all block heights are stored with the correct amount of bytes
```

Comment:
nit: Maybe replace this comment with just using `.expect(...)` instead of `.unwrap()`